aws/table_aws_macie2_classification_job.go

```diff
@@ -191,6 +206,13 @@ func getMacie2ClassificationJob(ctx context.Context, d *plugin.QueryData, h *plu
 	// Get call
 	op, err := svc.DescribeClassificationJob(params)
 	if err != nil {
+		if awsErr, ok := err.(awserr.Error); ok {
+			// Throws AccessDeniedException: Macie is not enabled. when AWS Macie is not enabled in a region
+			if awsErr.Message() == "Macie is not enabled." {
```

Comment:
@bigdatasourav Do you have a full output of what the error object looks like for reference?

Reply:
@cbruno10 It is available in the issue - https://github.com/turbot/steampipe-plugin-aws/issues/641